llvm/test/CodeGen/AArch64/variant-pcs.ll

```diff
@@ -2,6 +2,9 @@
 ; RUN: llc -mtriple=aarch64-linux-gnu -mattr=+sve -filetype=obj -o - %s \
 ; RUN:   | llvm-readobj --symbols - | FileCheck %s --check-prefix=CHECK-OBJ
 
+; Check we don't crash when using a Mach-O object format.
+; RUN: llc -mtriple=arm64-apple-macosx15.0.0 -mattr=+sve -filetype=obj -o /dev/null %s
```

Comment:
I'm not entirely sure how useful this RUN line is, because it's guarding against a case that should never have happened.

Reply:
Not sure I follow? All cases that cause a crash should never happen, but once fixed there should be a test to prevent a future regression.

Reply:
You're right, ignore my comment here, it seems like a valid case to test.